CourseGrab/Keys.swift

```diff
@@ -8,25 +8,14 @@
 
 import Foundation
 
-/**
- `Keys` is a list of expected hidden key value pairs in `Keys.plist`, used for sensitive
- developer information like API keys and secrets.
- */
-enum Keys: String {
-    case googleClientID = "google-client-id"
-
-    /**
-     `value` is the string representation of the key. Implicitly unwrapped because
-     the app should crash without Keys stored properly, while allowing you to check
-     for nil with `value == nil`.
-     */
-    var value: String! {
-        return Keys.keyDict[rawValue] as? String
-    }
+struct Keys {
```

Comment:
I would be more descriptive and call this `SecretKeys` or `PrivateKeys`.

Reply:
Let's keep it for consistency with other projects for the time being. We can change it down the road

Reply:
I guess that's reasonable. Granted, we did already make a lot of changes, and I think it's valid to break convention if you have a better alternative. I guess for now its fine and we can reconsider later.